pylint_odoo/test_repo/broken_module/models/broken_model.py

```diff
@@ -245,6 +245,23 @@ def my_method1(self, variable1):
         error_msg = _('Variable not translatable: {}'.format(variable1))
         error_msg = _('Variables not translatable: {}, {variable2}'.format(
             variable1, variable2=variable2))
+
+        # string with parameters without name
+        # so you can't change the order in the translation
+        _('%s %d') % ('hello', 3)
+        _('%s %s') % ('hello', 'world')
+        _('%s %s %s') % ('hello', 'world!')
+
+        # Valid cases
+        _('{0} {1}') % ('hello', 3)
```

Comment:
Is it worth it?

Reply:
I think a few examples is enough. I removed a few examples.

Reply:
This line is not a valid case. It uses str-format syntax with `%`

Reply:
You right, I removed it.